bot/exts/evergreen/pythonfacts.py

```diff
@@ -0,0 +1,31 @@
+import itertools
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+with open('bot/resources/evergreen/python_facts.txt') as file:
+    FACTS = list(file)
+    FACT_CYCLE = itertools.cycle(FACTS)
+
+COLORS = [Colours.python_blue, Colours.python_yellow]
+COLOR_CYCLE = itertools.cycle(COLORS)
```

Comment:
Since `COLORS` is not being used anywhere, directly put it in the cycle, same for the facts.

Reply:
Pardon?

Reply:
```py
 COLOR_CYCLE = itertools.cycle([Colours.python_blue, Colours.python_yellow])
```